sway-lib-std/src/token.sw

```diff
@@ -12,6 +12,29 @@ use ::outputs::{Output, output_amount, output_count, output_type};
 /// Mint `amount` coins of the current contract's `asset_id` and transfer them
 /// to `to` by calling either force_transfer_to_contract() or
 /// transfer_to_address(), depending on the type of `Identity`.
+///
+/// # WARNING
+///
+/// This will transfer coins to a contract even with no way to retrieve them
```

Comment:
We should probably add an "if" here somewhere saying that if the receiver is a contract, then ...